src/test/ui/lifetimes/re-empty-in-error.stderr

```diff
@@ -4,7 +4,7 @@ error: higher-ranked lifetime error
 LL |     foo(&10);
    |     ^^^^^^^^
    |
-   = note: could not prove `for<'b, 'a> &'b (): 'a`
+   = note: could not prove `for<'b> &'b (): '_`
```

Comment:
With the proposed change we don't create names for all anonymous late-bound vars.

Reply:
Yeah, this is what I did locally. Kind of weird. I guess it makes more sense to do what you had before.